.github/actions/sbom-update/action.yml

```diff
@@ -0,0 +1,27 @@
+name: Generate SBOM
```

Comment:
auto-opening a PR after another PR merges is just likely to be forgotten by our team.  Could we instead add a PR check that blocks a PR from merging if there is a diff in the generated SBOM (and include tooling so that we can generate the SBOM locally)?

This does requires manual generation from us but this works better for our workflow.